src/parseOption.js

```diff
@@ -0,0 +1,8 @@
+const parseOption = (optionDescription) => {
```

Comment:
I hope there is no need to create separate modules and files for each function. You can export an object as well:
```javascript
const parseOption = () => {};
const parseSection = () => {};

module.exports = {
   parseOption: parseOption,
   parseSection: parseSection
};

```

Reply:
I suggest using classes as well, JS supports not the only [procedural paradigm](https://en.wikipedia.org/wiki/Procedural_programming).